lib/salus/package_utils/package_lock_json.rb

```diff
@@ -56,6 +56,8 @@ def record_dep_locations
 
     # recursively store all dependency names as keys
     def get_dep_names(data)
+      return unless data.key?("dependencies")
```

Comment:
It would be great if we can have a unit test related the changes in 'def get_dep_names'.

We can add the unit test in a separate pr.

It's not recommended to use get_* or set_* at the beginning of method name in ruby? Can we use a method name like 'def generate_dep_names_for(data)'.

We can make these changes in a separate pr.